docs/docs/external/getting-started/interop/asset-transfer.md

```diff
@@ -14,11 +14,13 @@ pagination_next: external/getting-started/enabling-weaver-network/overview
 This document lists sample ways in which you can exercise the asset-transfer interoperation protocol on the test network [launched earlier](../test-network/overview).
 
 Once the networks, relays, and drivers have been launched, and the ledgers bootstrapped, you can trigger the following interoperation flows corresponding to distinct asset-sharing combinations _other combinations of DLTs will be supported soon_):
-1. **Fabric with Fabric**: One Fabric network transfers either a bond or some tokens owned by Alice to Bob in the other network
+## 1. Fabric with Fabric
+
+One Fabric network transfers either a bond or some tokens owned by Alice to Bob in the other network
 
 Assuming that the `simpleassettransfer` chaincode has been deployed in both networks, run the following steps by navigating to the `samples/fabric/fabric-cli` folder (_the Go CLI doesn't support asset transfer yet_).
 
-## Transfer or recover a bond (non-fungible) asset
+### **Transfer or recover a bond (non-fungible) asset**
```

Comment:
Can we avoid bold here in all sub headings, they are anyways in bold, Right column list of subheadings seems strange due to this.